xsd/netex_framework/netex_reusableComponents/netex_facility_version.xsd

```diff
@@ -518,6 +528,74 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 		</xsd:sequence>
 	</xsd:group>
+	<!-- ==== GroupOfFacilities=======================================-->
+	<xsd:complexType name="groupsOfFacilites_RelStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for list of  GROUP OF FACILITies.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="containmentAggregationStructure">
+				<xsd:sequence>
+					<xsd:element ref="GroupOfFacilities" maxOccurs="unbounded"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:element name="GroupOfFacilities" abstract="false" substitutionGroup="GroupOfEntities">
+		<xsd:annotation>
+			<xsd:documentation>A grouping of FACILITies which will be commonly referenced for a specific purpose. +v2.0</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:restriction base="GroupOfFacilities_VersionStructure">
+					<xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="EntityInVersionGroup" minOccurs="0"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="DataManagedObjectGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="GroupOfEntitiesGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="GroupOfFacilitiesGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="GroupOfFacilitiesIdType"/>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="GroupOfFacilities_VersionStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for GROUP OF FACILITies.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="GroupOfEntities_VersionStructure">
+				<xsd:sequence>
+					<xsd:group ref="GroupOfFacilitiesGroup"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="GroupOfFacilitiesGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for GROUP OF FACILITies.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="PublicCode" type="ExternalObjectRefStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Public code for GROUP OF SITES.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="members" type="facilitiesRefs_RelStructure" minOccurs="0">
```

Comment:
I don't understand here: we have a _facilitiesRefs_RelStructure_ (_facilitiesRef_ ...) but the Facility does not exist as a first class object (no Facility element), so nothing to refer to at the end of the day !
And defining a Facility element is not really possible since it a Facility is basically an enumeration...

The only way to go that I can see is members being SiteFacilitySets and/or ServiceFacilitySets

Reply:
done differently